src/app/components/profile/profile.component.ts

```diff
@@ -120,11 +115,16 @@ export class ProfileComponent implements OnInit {
     const TOKEN_INPUT = this.globalService.formItemForLabel(this.formTokenComponents, 'token');
     TOKEN_INPUT.type = this.tokenModalButtonFlag ? 'password' : 'text';
   }
-
   /**
    * Displays a Modal to update user details
    */
   updateUserDetails() {
+    this.first_name = (this.user['first_name'] === '-') ? '' : this.user['first_name'];
+    this.last_name = (this.user['last_name'] === '-') ? '' : this.user['last_name'];
+    this.affiliation = (this.user['affiliation'] === '-') ? '' : this.user['affiliation'];
+    this.google_scholar_url = (this.user['google_scholar_url'] === '-') ? '' : this.user['google_scholar_url'];
+    this.github_url = (this.user['github_url'] === '-') ? '' : this.user['github_url'];
+    this.linkedin_url = (this.user['linkedin_url'] === '-') ? '' : this.user['linkedin_url'];
```

Comment:
As these variables are not used again so better not to use them as component variable instead simply we can use them as `const` variable.

Reply:
ok